src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -92,6 +93,7 @@ export default defineComponent({
       title: '',
       channelName: null,
       channelId: null,
+      channelThumbnail: null,
```

Comment:
Not PR specific, I got this opinion for the entire app
`channelThumbnail` always looks like a variable with an object to me
```suggestion
      channelThumbnailUrl: null,
```